Backend linters

nickbeaird opened the pull request:
Add in backend linter formatting rules to be verified by the team. 

This PR adds in linter and prettier formatting in hopes to be approved the the dev team. A follow up PR will have to be made to     enact the changes presented, which will be easy with the settings made here.

nickbeaird committed:
Add linter rules for backend

nickbeaird committed:
Setup prettier to run in the backend

clockwerkz reviewed (approved):
Digging it! We definitely have been running too long with any sort of standardized tooling with regards to formatting and linting. Thanks for taking care of this.
Extra points for setting the tabWidth to 2 spaces instead of 4!

nickbeaird committed:
Update yarn lock file

nickbeaird committed:
Fix quotes in prettier config

nickbeaird committed:
Make script name unique

- Having the script name the same as the commands causing both to run
  when run on the command line.
